### PR TITLE
Disable single table mode in API

### PR DESCRIPTION
**Pull request by niksv**
Related to #5313

### CODE REVIEW

File: src-java/nbworker-topology/nbworker-storm-topology/src/main/java/org/openkilda/wfm/topology/nbworker/services/KildaConfigurationService.java [hunk 1]
```diff
@@ -51,6 +51,10 @@ public KildaConfiguration getKildaConfiguration() {
      */
     public KildaConfiguration updateKildaConfiguration(KildaConfiguration kildaConfiguration) {
         log.info("Process kilda config update - config: {}", kildaConfiguration);
+        if (Boolean.FALSE.equals(kildaConfiguration.getUseMultiTable())) {
```
Comment (pkazlenka) [inline src-java/nbworker-topology/nbworker-storm-topology/src/main/java/org/openkilda/wfm/topology/nbworker/services/KildaConfigurationService.java:54]: Just curious, what is this trick with `Boolean..equals()` needed for?

File: src-java/nbworker-topology/nbworker-storm-topology/src/main/java/org/openkilda/wfm/topology/nbworker/services/KildaConfigurationService.java [hunk 2]
```diff
@@ -51,6 +51,10 @@ public KildaConfiguration getKildaConfiguration() {
...
+            throw new IllegalArgumentException("Single table mode was deprecated. OpenKilda doesn't support it "
+                    + "anymore. Configuration property `use_multi_table` can't has 'false' value.");
```
Comment (pkazlenka) [inline src-java/nbworker-topology/nbworker-storm-topology/src/main/java/org/openkilda/wfm/topology/nbworker/services/KildaConfigurationService.java:56]: Mistake - `can't have`. Or, even better, `can't be 'false'`.

File: src-java/nbworker-topology/nbworker-storm-topology/src/test/java/org/openkilda/wfm/topology/nbworker/services/SwitchOperationsServiceTest.java [hunk 3]
```diff
@@ -137,7 +137,7 @@ public void islBfdPropertiesChanged(Endpoint source, Endpoint destination) {
     }
 
     @Test
-    public void shouldUpdateLinkUnderMaintenanceFlag() throws SwitchNotFoundException {
+    public void updateLinkUnderMaintenanceFlag() throws SwitchNotFoundException {
```
Comment (pkazlenka) [inline src-java/nbworker-topology/nbworker-storm-topology/src/test/java/org/openkilda/wfm/topology/nbworker/services/SwitchOperationsServiceTest.java:140]: Aren't you afraid of solving bunch of conflicts between this PR and the ones with migration to Junit5? May be it's better to move this renaming to migration PR?

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ConnectedDevicesSpec.groovy [hunk 4]
```diff
@@ -994,79 +988,6 @@ srcDevices=#newSrcEnabled, dstDevices=#newDstEnabled"() {
         descr = "default(no-vlan) flow endpoints on ${getDescr(srcDefault, dstDefault)}"
     }
 
-    @Tidy
```
Comment (pkazlenka) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ConnectedDevicesSpec.groovy:997]: Fuck yeah!

**Review by pkazlenka — APPROVED** [review]
Very nice. Just fix misprint.

File: src-java/nbworker-topology/nbworker-storm-topology/src/main/java/org/openkilda/wfm/topology/nbworker/services/KildaConfigurationService.java (hunk 1 above)
Comment (niksv) [reply src-java/nbworker-topology/nbworker-storm-topology/src/main/java/org/openkilda/wfm/topology/nbworker/services/KildaConfigurationService.java:54]: kildaConfiguration.getUseMultiTable() can be `null` and it's okay. 
But I need to check that the value is not false. 
If I will use `if (!kildaConfiguration.getUseMultiTable())` I can get NPE

File: src-java/nbworker-topology/nbworker-storm-topology/src/main/java/org/openkilda/wfm/topology/nbworker/services/KildaConfigurationService.java (hunk 2 above)
Comment (niksv) [reply src-java/nbworker-topology/nbworker-storm-topology/src/main/java/org/openkilda/wfm/topology/nbworker/services/KildaConfigurationService.java:56]: fixed

File: src-java/nbworker-topology/nbworker-storm-topology/src/test/java/org/openkilda/wfm/topology/nbworker/services/SwitchOperationsServiceTest.java (hunk 3 above)
Comment (niksv) [reply src-java/nbworker-topology/nbworker-storm-topology/src/test/java/org/openkilda/wfm/topology/nbworker/services/SwitchOperationsServiceTest.java:140]: The base of this PR is Junit migration patch, so This PR will be merged after Junit

**Comment by artHalMirantis** [comment]
all in one just enough to test it
will create separate task to remove single-table tests from TAF

**Review by IvanChupin — APPROVED** [review]
Great job!

**Pull request merged**